source/AI.cpp

```diff
@@ -2714,9 +2714,44 @@ void AI::DoSurveillance(Ship &ship, Command &command, shared_ptr<Ship> &target)
 		unsigned total = targetShips.size() + targetPlanets.size() + targetSystems.size();
 		if(!total)
 		{
-			// If there is nothing for this ship to scan, have it hold still
-			// instead of drifting away from the system center.
-			Stop(ship, command);
+			// If there is nothing for this ship to scan, have it patrol the entire system
+			// instead of drifting or stopping.
+			// Also allows the ship to land.
+			double radius = 1000. * 1000. * 1.1;
```

Comment:
where does this come from? shouldnt we use the range limit we made into a gamerule?

Reply:
~~Probably, I just don't know how they work.~~

Reply:
oh nvm it's not a gamerule it's per system